packages/components/src/templates/next/components/internal/Siderail/SiderailLabel.tsx

```diff
@@ -0,0 +1,56 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { tv } from "tailwind-variants"
+
+import type { Item } from "./types"
+import { useSiderailContext } from "./SiderailContext"
+
+const createSiderailLabelStyles = tv({
+  slots: {
+    container: "group px-3 py-4 transition hover:text-brand-interaction",
+    icon: "-mt-0.5 inline h-6 w-6 opacity-0 transition group-hover:translate-x-1 group-hover:opacity-100",
```

Comment:
issue: eh I think it got worse

<img width="278" alt="image" src="https://github.com/user-attachments/assets/cecc8ec4-bcb3-46dd-9930-fdc535b77064">

Reply:
expected right? now the icon sits with the line and breaks properly. that one just nice breakpoint ma. @sehyunidaaa right

Reply:
oh errr shouldn't they be in a row! so the arrow is where the minus is and the text should wrap

Reply:
Omg so was always justify-between???? Ok

Reply:
ya my bad i never prototype the multi line version

Reply:
done